src/menu/viewRole.ts

```diff
@@ -43,6 +43,7 @@ export const projectViewRole = (
       { type: 'separator' },
       { role: 'minimize' },
       { role: 'zoom' },
+      { role: 'toggleDevTools' },
```

Comment:
We still need this to debug the production build and this seemed like a better location to me.